build/css/main.css

```diff
@@ -16142,39 +16142,129 @@ textarea.form-control-lg {
 }
-.govcy-form .govcy-text-input-error, .govcy-form .govcy-text-area-error {
+.govcy-form .govcy-form-control .govcy-text-input-error, .govcy-form .govcy-form-control .govcy-text-area-error {
   border-color: #D4351C;
 }
-.govcy-form .govcy-text-input-char_20, .govcy-form .govcy-text-area-char_20 {
+.govcy-form .govcy-form-control .govcy-text-input-char_20, .govcy-form .govcy-form-control .govcy-text-area-char_20 {
   width: 14.16rem;
 }
-.govcy-form .govcy-text-input-char_10, .govcy-form .govcy-text-area-char_10 {
+.govcy-form .govcy-form-control .govcy-text-input-char_10, .govcy-form .govcy-form-control .govcy-text-area-char_10 {
   width: 7.8rem;
 }
-.govcy-form .govcy-text-input-char_5, .govcy-form .govcy-text-area-char_5 {
+.govcy-form .govcy-form-control .govcy-text-input-char_5, .govcy-form .govcy-form-control .govcy-text-area-char_5 {
   width: 4.2rem;
 }
-.govcy-form .govcy-text-input-char_4, .govcy-form .govcy-text-area-char_4 {
+.govcy-form .govcy-form-control .govcy-text-input-char_4, .govcy-form .govcy-form-control .govcy-text-area-char_4 {
   width: 3.8rem;
 }
-.govcy-form .govcy-text-input-char_3, .govcy-form .govcy-text-area-char_3 {
+.govcy-form .govcy-form-control .govcy-text-input-char_3, .govcy-form .govcy-form-control .govcy-text-area-char_3 {
   width: 3.1rem;
 }
-.govcy-form .govcy-text-input-char_2, .govcy-form .govcy-text-area-char_2 {
+.govcy-form .govcy-form-control .govcy-text-input-char_2, .govcy-form .govcy-form-control .govcy-text-area-char_2 {
   width: 2.2rem;
 }
+.govcy-form .govcy-form-control .govcy-checkbox,
+.govcy-form .govcy-form-control .govcy-radio-btn {
+  display: block;
+  position: relative;
+  padding: 6px 16px 8px 60px;
+  margin-top: 12px;
+  cursor: pointer;
+  font-size: 22px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+.govcy-form .govcy-form-control .govcy-checkbox .govcy-checkbox-input, .govcy-form .govcy-form-control .govcy-checkbox .govcy-radio-input,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-checkbox-input,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-radio-input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+.govcy-form .govcy-form-control .govcy-checkbox .govcy-checkbox-input:checked ~ .govcy-tickbox, .govcy-form .govcy-form-control .govcy-checkbox .govcy-checkbox-input:checked ~ .govcy-radio, .govcy-form .govcy-form-control .govcy-checkbox .govcy-radio-input:checked ~ .govcy-tickbox, .govcy-form .govcy-form-control .govcy-checkbox .govcy-radio-input:checked ~ .govcy-radio,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-checkbox-input:checked ~ .govcy-tickbox,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-checkbox-input:checked ~ .govcy-radio,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-radio-input:checked ~ .govcy-tickbox,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-radio-input:checked ~ .govcy-radio {
```

Comment:
The outline or box-shadow, should be on `:focus` and maybe `:active` state not `checked`. 

This is consistent with the active state of text, textarea and button active state, so when the input is in focus it will look the same. Also if you travel on the page using the 'Tab' key, right now there is nothing that visually shows on which checkbox or radio you are a the moment.

https://docs.google.com/document/d/1C2mCPbZ6ogzxC3p3n6M8sbxxi85yll2TI4dUdJ3Wsxc/edit#heading=h.qu3tejaz5qfg